operatorconfig/driverconfig/powerstore/v2.4.0/controller.yaml

```diff
@@ -227,7 +227,7 @@ spec:
             - name: X_CSI_DRIVER_NAME
               value: "csi-powerstore.dellemc.com"
             - name: X_CSI_POWERSTORE_EXTERNAL_ACCESS
-              value: "None"
+              value: <X_CSI_POWERSTORE_EXTERNAL_ACCESS>
```

Comment:
@karthikk92 , please check if quotes are required or not.. if it is not required please remove for the other environment variables.. Let us be consistent

Reply:
Quotes are not required here since driver expects without quote and expects empty value. It wont work while testing. So removed quote. Here if quotes are passed it will take as string and expects ip. For other since its required its added , for example for external health monitor its expected with quote "false" or "true" as we can see before configurable parameter this is hard coded with value "false"(with quote) .